app/controllers/idv/in_person/address_controller.rb

```diff
@@ -16,6 +17,8 @@ def show
       end
 
       def update
+        # don't clear the ssn when updating address, clear after SsnController
```

Comment:
Thanks for leaving comment- I understand b/c we talked about this but think it will be helpful to future devs